build_in_docker.sh

```diff
@@ -0,0 +1,10 @@
+container=$(docker run --rm -it -d ubuntu-dev bash)
+name=$(python setup.py sdist | grep removing | cut -d' ' -f2 | sed "s/'//g")
+docker cp dist/${name}.tar.gz ${container}:/
+docker exec ${container} tar xf /${name}.tar.gz
+docker cp debian ${container}:/${name}
+#docker exec ${container} sh -c "python3 -m venv venv && venv/bin/pip install -r ${name}/requirements.txt"
+docker exec ${container} sh -c "cd ${name} && dpkg-buildpackage -us -uc -b && mkdir /dist && cp ../*.deb /dist"
+docker cp ${container}:/dist .
+docker exec -it ${container} bash
+docker container stop ${container}
```

Comment:
I am not familiar with docker. this is creating an ubuntu container for the qualcoder software? Is is similar for OSX and Windows?

Reply:
I think it is not easy possible for Windows or Mac, not sure though. However you can spin up a virtualmachine if you want.